test/modules/multi-image.js

```diff
@@ -189,4 +189,22 @@
         ]);
     });
 
+    asyncTest('Viewer.addSimpleImage', function() {
+        viewer.addHandler("open", function openHandler() {
+            viewer.removeHandler("open", openHandler);
+
+            viewer.world.addHandler('add-item', function itemAdded(event) {
+                viewer.world.removeHandler('add-item', itemAdded);
+                equal(event.item.opacity, 0.5, 'Opacity option not set using addSimpleImage');
```

Comment:
Test message should be written in terms of what is expected rather than what failed. For instance, this could be "Opacity option set using addSimpleImage".